symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/util/PresentationMLParser.java

```diff
@@ -0,0 +1,60 @@
+package com.symphony.bdk.core.service.message.util;
+
+import com.symphony.bdk.core.service.message.exception.PresentationMLParserException;
+
+import lombok.Generated;
+import lombok.SneakyThrows;
+import org.apiguardian.api.API;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+/**
+ * Helper class for parsing the PresentationML to text content.
+ */
+@API(status = API.Status.STABLE)
+public class PresentationMLParser {
```

Comment:
If we want to stick to CamelCase it should be PresentationMl right? but maybe here we can consider it a special name and not an acronym

Reply:
I'd keep `ML` in uppercase just for `MessageML` and `PresentationML` (widely written this way in doc https://developers.symphony.com/symphony-developer/docs/presentationml)

Reply:
Let's keep like that then!